tests/lib/snaps/test-snapd-check-fs-access/meta/snap.yaml

```diff
@@ -1,18 +0,0 @@
-name: test-snapd-check-fs-access
```

Comment:
Is this snap not used by any test?

Reply:
I removed the uses, it was being used by 2 tests